src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1752,7 +1752,12 @@ private bool CheckEventValueKind(BoundEventAccess boundEvent, BindValueKind valu
                         // NOTE: availability of well-known members is checked in BindEventAssignment because
                         // we don't have the context to determine whether addition or subtraction is being performed.
 
-                        if (ReportUseSiteDiagnostics(eventSymbol, diagnostics, eventSyntax))
+                        if (receiver?.Kind == BoundKind.BaseReference && eventSymbol.IsAbstract)
+                        {
+                            Error(diagnostics, ErrorCode.ERR_AbstractBaseCall, boundEvent.Syntax, eventSymbol);
```

Comment:
I see -- we have no shared hierarchy between properties and events and this check is currently performed in CheckPropertyValueKind for properties.

In that case, this LGTM.